hw3/form.py

```diff
@@ -0,0 +1,10 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField
+from wtforms.validators import DataRequired, Email, Length
+
+
+class RegistrationForm(FlaskForm):
+    firstname = StringField('firstname', validators=[DataRequired()])
+    lastname = StringField('lastname', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(), Email()])
+    password = PasswordField('Password', validators=[DataRequired(), Length(min=6)])
```

Comment:
Если добавить поле `SubmitField`.
```suggestion
    password = PasswordField('Password', validators=[DataRequired(), Length(min=6)])
    submit - SubmitField('Зарегистрироваться')
```